service/src/main/java/org/kiwiproject/champagne/model/Component.java

```diff
@@ -0,0 +1,29 @@
+package org.kiwiproject.champagne.model;
+
+import java.time.Instant;
+
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * Representation of a deployable component in the system.
+ */
+@Builder
+@Getter
+public class Component {
```

Comment:
Should this and `Host` be value classes? i.e. do they need, or would it be useful to have, `equals`, `hashCode`, `toString`?